config/config_helpers.go

```diff
@@ -63,19 +64,19 @@ type helperFunction struct {
 	returnType cty.Type
 }
 
-func (context *resolveContext) getHelperFunctions() map[string]helperFunction {
+func (ctx *resolveContext) getHelperFunctions() map[string]helperFunction {
```

Comment:
context is used to refers the standard go library `context` that is used by aws-sdk-go-v2.